components/ActionBox/components/commons.tsx

```diff
@@ -0,0 +1,140 @@
+import styled, { css } from 'styled-components';
```

Comment:
nit pick. file name -> `common` instead of `commons`

Reply:
+1

Reply:
Done